phpBB/adm/style/acp_profile.html

```diff
@@ -17,7 +17,7 @@
 		</div>
 	<!-- ENDIF -->
 
-	<form id="add_profile_field" method="post" action="{U_ACTION}">
+	<form id="add_profile_field" method="post" action="{U_ACTION}"{S_FORM_ENCTYPE}>
```

Comment:
What I meant is: `<!-- IF S_FORM_ENCTYPE is defined and S_FORM_ENCTYPE is not empty --> enctype="{S_FORM_ENCTYPE}"<!-- ENDIF -->`

Thanks